devtools-integration-tests/index.json

```diff
@@ -1,6 +1,44 @@
 [
     {
         "name": "Testerina Integration Tests",
-        "path": "testerina-integration-tests"
+        "path": "testerina-integration-tests",
+        "module": "testerina-object-mocking-test",
+        "cmd": "test"
+    },
+    {
+        "name": "Packerina Command Tests",
```

Comment:
Is the 'name' value being used anywhere? If not shall we remove it?

Reply:
No. We can remove that.